aten/src/ATen/native/mps/OperationUtils.mm

```diff
@@ -142,7 +145,7 @@ MPSDataType getMPSScalarType(ScalarType scalar_type) {
     case ScalarType::Bool:
       return MPSDataTypeBool;
     default:
-      TORCH_INTERNAL_ASSERT(false, "Trying to convert ", scalar_type, " to the MPS backend but there is no mapping for it.")
+      TORCH_INTERNAL_ASSERT(false, "Trying to convert ", scalar_type, " to the MPS backend but that backend doesn't exist.")
```

Comment:
I don't think this is the right phrasing. We can just say that `but MPS backend doesn't have support for it`

Reply:
This is an internal assert as all the dtypes should be handled above.
The only way for this to trigger is if someone adds a new dtype but does not update this.
But I guess there are weird quantized dtypes that we don't support.
Let me change that back to a non-internal then and use your proposed phrasing.

Reply:
This is an internal assert as all the dtypes should be handled above.
The only way for this to trigger is if someone adds a new dtype but does not update this.
But I guess there are weird quantized dtypes that we don't support.
Let me change that back to a non-internal then and use your proposed phrasing.